Atari-Game-Player/steps/build_dqn.py

```diff
@@ -0,0 +1,29 @@
+from .configs import PreTrainingConfigs
+from dqn.model import *
+from zenml.steps import step, Output
+
+
+@step
+def build_dqn(
+    config: PreTrainingConfigs, game_wrapper: GameWrapper
+) -> Output(MAIN_DQN=tf.keras.Model, target_dqn=tf.keras.Model):
+    """
+    It builds the main and target DQN.
+    
+    :param config: PreTrainingConfigs
+    :type config: PreTrainingConfigs
+    :param game_wrapper: The GameWrapper object that wraps the Atari game
+    :type game_wrapper: GameWrapper
+    :return: the main DQN and the target DQN.

```

Comment:
I would also go for google style docstrings here : https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings